src/libs/ReportPrimaryActionUtils.ts

```diff
@@ -71,17 +72,30 @@ function isPayAction(report: Report, policy: Policy) {
     }
 
     const isProcessingReport = isProcessingReportUtils(report);
+
+    if (!isProcessingReport) {
+        return false;
+    }
+
     const isInvoiceReport = isInvoiceReportUtils(report);
+
+    if (isInvoiceReport) {
+        return true;
+    }
     const isIOUReport = isIOUReportUtils(report);
 
-    if ((isInvoiceReport || isIOUReport) && isProcessingReport) {
+    if (isIOUReport && isReportPayer) {
         return true;
     }
 
     return false;
 }
 
-function isExportAction(report: Report, policy: Policy) {
+function isExportAction(report: Report, policy?: Policy) {
+    if (!policy) {
+        return false;
+    }
```

Comment:
@luacmartins here I assumed that it's not possible to export report if there is no policy, for example for 1:1 IOU

Reply:
That's correct